server.cc

```diff
@@ -1,265 +1,91 @@
-                bool close) {
-    boost::ignore_unused(bytes_transferred);
-
-    if (ec) return fail(ec, "write");
-
-    if (close) {
-      // This means we should close the connection, usually because
-      // the response indicated the "Connection: close" semantic.
-      return do_close();
-    }
-
-    // We're done with the response so delete it
-    res_ = nullptr;
-
-    // Read another request
-    do_read();
-  }
-
-  void do_close() {
-    // Send a TCP shutdown
-    boost::system::error_code ec;
-    socket_.shutdown(tcp::socket::shutdown_send, ec);
-  }
-};
-
-// Accepts incoming connections and launches the sessions
-class listener : public std::enable_shared_from_this<listener> {
-  tcp::acceptor acceptor_;
-  tcp::socket socket_;
-
- public:
-  listener(boost::asio::io_context& ioc, tcp::endpoint endpoint)
-      : acceptor_(ioc), socket_(ioc) {
-    boost::system::error_code ec;
-
-    // Open the acceptor
-    acceptor_.open(endpoint.protocol(), ec);
-    if (ec) {
-      fail(ec, "open");
-      return;
-    }
-
-    // Allow address reuse
-    acceptor_.set_option(boost::asio::socket_base::reuse_address(true), ec);
-    if (ec) {
-      fail(ec, "set_option");
-      return;
-    }
-
-    // Bind to the server address
-    acceptor_.bind(endpoint, ec);
-    if (ec) {
-      fail(ec, "bind");
-      return;
-    }
-
-    // Start listening for connections
-    acceptor_.listen(boost::asio::socket_base::max_listen_connections, ec);
-    if (ec) {
-      fail(ec, "listen");
-      return;
-    }
-  }
-
-  // Start accepting incoming connections
-  void run() {
-    if (!acceptor_.is_open()) return;
-    do_accept();
-  }
-
-  void do_accept() {
-    acceptor_.async_accept(socket_,
-                           std::bind(&listener::on_accept, shared_from_this(),
-                                     std::placeholders::_1));
-  }
-
-  void on_accept(boost::system::error_code ec) {
-    if (ec) {
-      fail(ec, "accept");
-    } else {
-      // Create the session and run it
-      std::make_shared<session>(std::move(socket_))->run();
-    }
-
-    // Accept another connection
-    do_accept();
-  }
-};
-
-int main(int argc, char* argv[]) {
-  auto const address = boost::asio::ip::make_address("127.0.0.1");
-  auto const port = static_cast<uint16_t>(std::atoi("8888"));
-  auto const threads = std::max<int>(1, std::atoi("16"));
-
-  // The io_context is required for all I/O
-  boost::asio::io_context ioc{threads};
-
-  // Create and launch a listening port
-  std::make_shared<listener>(ioc, tcp::endpoint{address, port})->run();
-  std::cout << "Server started..."
-            << "\n";
-
-  // Run the I/O service on the requested number of threads
-  std::vector<std::thread> v;
-  v.reserve(threads - 1);
-  for (auto i = threads - 1; i > 0; --i) v.emplace_back([&ioc] { ioc.run(); });
-  ioc.run();
-
-  return EXIT_SUCCESS;
+#include <served/plugins.hpp>
+#include <served/served.hpp>
+#include "./apis.h"
```

Comment:
The prefix `./` looks redundant. Can you eliminate it?

Reply:
This is required for cpplint:
```
Include the directory when naming .h files  [build/include] [4]
```